File: frontend/src/app/models/privacy-policy.ts
export interface PrivacyPolicy {
  id: number;
  version: number;
  title: string;
  content: string;
  publishedAtUtc: string;
  isCurrent: boolean;
  publishedBy?: string | null;
}

export interface PrivacyPolicyNotice {
  hasUpdate: boolean;
  currentId?: number | null;
  currentVersion?: number | null;
  currentPublishedAtUtc?: string | null;
}

File: TodoApi/Models/PrivacyPolicy/PrivacyPolicy.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace TodoApi.Models.PrivacyPolicy
{
    public class PrivacyPolicy
    {
        public int Id { get; set; }

        public int Version { get; set; }

        [Required]
        [MaxLength(200)]
        public string Title { get; set; } = null!;

        [Required]
        public string Content { get; set; } = null!;

        public DateTime PublishedAtUtc { get; set; }

        public bool IsCurrent { get; set; }

        [MaxLength(200)]
        public string? PublishedBy { get; set; }
    }
}
